config/theme-tabler.php

```diff
@@ -19,22 +19,10 @@
      * Simple pick a layout and let Backpack decide the best look for it.
      * No extra step is required.
      *
-     * Where should the menu be positioned?
+     * Possible values: horizontal, horizontal_dark, horizontal_overlap, vertical,
+     * vertical_dark, vertical_transparent, right_vertical, right_vertical_dark, right_vertical_transparent
      */
     'layout' => 'horizontal',
-    // horizontal
-    // horizontal_dark
-    // horizontal_overlap
```

Comment:
I think `navbar_overlap` isn't as pretty when the topbar has two rows:

<img width="1512" alt="CleanShot 2023-04-19 at 11 31 05@2x" src="https://user-images.githubusercontent.com/1032474/233016831-48faacca-9518-4667-8353-42a78495815a.png">

I find it much prettier when it only has one row:
<img width="1512" alt="CleanShot 2023-04-19 at 11 31 36@2x" src="https://user-images.githubusercontent.com/1032474/233016978-2ddf24b0-1182-4678-a157-181427489da8.png">

But we'd have a problem there in the demo, because we have WAAAY too many level-1 menu items. But we can probably slim that down by:
- moving Other entities to Monsters & Stuff (they are _stuff_ after all)
- moving everything that is provided by a Backpack addon to a new Addons item

But even without the above... I still think we should have a layout that only has one topmenu line, not two. Because a well-organised admin panel shouldn't have more than 7-8 first-level items in the menu. And `navbar-overlap`... that's a good option to introduce it.